### PR TITLE
US593021: Updated to run on Java 17

### PR DESCRIPTION
**Pull request by michael-bryson**
https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=593021

### CODE REVIEW

File: release-notes-2.0.0.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+!not-ready-for-release!
+
+#### Version Number
+${version-number}
+
+#### Breaking Changes
+- US593021: Nashorn script engine is no longer supported.
```
Comment (dermot-hardy) [inline release-notes-2.0.0.md]: Unless I'm missing something this is not a breaking change for the autoscaler.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

File: release-notes-2.0.0.md (hunk 1 above)
Comment (michael-bryson) [reply release-notes-2.0.0.md]: I didn't think so but they are listed under Apollo Workers for fortify. If that is only for fortify and not relevant then I can revert the version change.

Comment (dermot-hardy) [reply release-notes-2.0.0.md]: Yes it's just for Fortify.

Comment (michael-bryson) [reply release-notes-2.0.0.md]: Reverted.

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**